mlflow/R/mlflow/R/model-registry.R

```diff
@@ -0,0 +1,47 @@
+#' Create registered model
+#'
+#' Creates a new registered model in the model registry
+#'
+#' @param name The name of the model to create.
+#' @param tags Additional metadata for the registered model (Optional).
+#' @param description Description for the registered model (Optional).
+#' @template roxlate-client
+#' @export
+mlflow_create_registered_model <- function(name, tags = NULL, description = NULL, client = NULL) {
+  client <- resolve_client(client)
+
+  response <- mlflow_rest(
+    "registered-models",
+    "create",
+    client = client,
+    verb = "POST",
+    version = "2.0/preview",
```

Comment:
I'm not sure we need the preview routes here

Reply:
I believe all the model registry endpoints still require the `preview` route: https://mlflow.org/docs/latest/rest-api.html#create-registeredmodel

Reply:
I think the documentation is outdated, it should point to the non-preview paths. Thanks for pointing that out @bramrodenburg!

Reply:
Ah ok! You're welcome :)

Removed the preview routes!